addons/website_livechat/tests/test_livechat_basic_flow.py

```diff
@@ -373,3 +374,18 @@ class TestLivechatBasicFlowHttpCaseMobile(HttpCaseWithUserDemo, TestLivechatComm
 
     def test_mobile_user_interaction(self):
         self.start_tour('/', 'im_livechat_request_chat_and_send_message', login=None)
+
+
+@tests.tagged("post_install", "-at_install")
+class TestWebsiteLivechatGetDiscussChannel(TestGetDiscussChannel):
```

Comment:
Why do we need this inheritance? There are already tests in the basic flow class above checking store data, can't we just add the test there? I don't like to have two classes on the same file. It holds for models or controllers but also for tests as it's cumbersome to find what you're looking for from the file name.